### PR TITLE
[doc] Add diagnostic tool section to GCP Dataproc getting started page [skip ci]

### PR DESCRIPTION
**Pull request by zhanga5**
which will be included from spark-rapids-user-tools package

Signed-off-by: Alex Zhang <alex4zhang@gmail.com>

### CODE REVIEW

**Comment by GaryShen2008** [comment]
LGTM，but should target to branch-22.10.

**Comment by zhanga5** [comment]
> LGTM，but should target to branch-22.10.

updated

**Review by mattahrens — COMMENTED** [review]
Couple general comments:

1. I would recommend adding a link to the Diagnostic tool in the quick start guide table of contents near the top, below this line: "* [Tune applications on GPU cluster](#tune-applications-on-gpu-cluster)"
2. In general, we want to refer to our solution in public documentation as the "RAPIDS Accelerator for Apache Spark" and not as "Spark RAPIDS".    So you can change that accordingly in a couple places.

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -127,6 +127,88 @@ Example output:
...
+*** Running diagnostic function "cuda_version" ***
+Connection to 34.171.177.95 closed.
+found cuda major version: 11
+*** Running diagnostic function "cuda_version" ***
+Connection to 35.188.25.57 closed.
+found cuda major version: 11
+*** Running diagnostic function "rapids_jar" ***
+Connection to 34.171.177.95 closed.
+found rapids jar version: 2.12-22.06.0
+...
```
[the remark this answers is not in the document]
Comment (GaryShen2008) [reply docs/get-started/getting-started-gcp.md:203]: That's a good idea. Let's discuss in the requirement doc.

File: docs/get-started/getting-started-gcp.md [hunk 2]
```diff
@@ -127,6 +127,88 @@ Example output:
 +--------------------------------+--------------------------------------------------+--------------------------------------------------------------------------------------------------+
 ```
 
+## Diagnose GPU Cluster
+
+Diagnostic tool can be run to check GPU cluster with Spark RAPIDS environment is healthy and
```
[the remark this answers is not in the document]
Comment (zhanga5) [reply docs/get-started/getting-started-gcp.md]: refined

File: docs/get-started/getting-started-gcp.md [hunk 3]
```diff
@@ -127,6 +127,88 @@ Example output:
...
+*** Running diagnostic function "cuda_version" ***
+Connection to 35.188.25.57 closed.
+found cuda major version: 11
+*** Running diagnostic function "rapids_jar" ***
+Connection to 34.171.177.95 closed.
+found rapids jar version: 2.12-22.06.0
+...
+```
+
+Please note, diagnostic tool supported
```
[the remark this answers is not in the document]
Comment (zhanga5) [reply docs/get-started/getting-started-gcp.md]: refined

**Comment by zhanga5** [comment]
> Couple general comments:
> 
> 1. I would recommend adding a link to the Diagnostic tool in the quick start guide table of contents near the top, below this line: "* [Tune applications on GPU cluster](#tune-applications-on-gpu-cluster)"
> 2. In general, we want to refer to our solution in public documentation as the "RAPIDS Accelerator for Apache Spark" and not as "Spark RAPIDS".    So you can change that accordingly in a couple places.

refined

File: docs/get-started/getting-started-gcp.md [hunk 4]
```diff
@@ -127,6 +128,89 @@ Example output:
 +--------------------------------+--------------------------------------------------+--------------------------------------------------------------------------------------------------+
 ```
 
+## Diagnose GPU Cluster
+
+The diagnostic tool can be run to check a GPU cluster with RAPIDS Accelerator for Apache Spark
+is healthy and ready for Spark jobs, such as installed Nvidia driver version, cuda-toolkit, RAPIDS
```
Comment (viadea) [inline docs/get-started/getting-started-gcp.md:134]:
```suggestion
is healthy and ready for Spark jobs, such as checking the version of installed NVIDIA driver version, cuda-toolkit, RAPIDS
```

File: docs/get-started/getting-started-gcp.md [hunk 5]
```diff
@@ -127,6 +128,89 @@ Example output:
...
+Accelerator and run some Spark test jobs etc. This tool also can be used by the frontline support
```
Comment (viadea) [inline docs/get-started/getting-started-gcp.md:135]:
```suggestion
Accelerator and running Spark test jobs etc. This tool also can be used by the frontline support
```

File: docs/get-started/getting-started-gcp.md [hunk 6]
```diff
@@ -127,6 +128,89 @@ Example output:
...
+found cuda major version: 11
+*** Running diagnostic function "rapids_jar" ***
+Connection to 34.171.177.95 closed.
+found rapids jar version: 2.12-22.06.0
+...
+```
+
+Please note that the diagnostic tool supports the following:
+
+* Dataproc 2.0 with image of Debian 10 or Ubuntu 18.04 (Rocky8 is coming soon)
```
Comment (viadea) [inline docs/get-started/getting-started-gcp.md:210]:
```suggestion
* Dataproc 2.0 with image of Debian 10 or Ubuntu 18.04 (Rocky8 support is coming soon)
```

**Review by viadea — CHANGES_REQUESTED** [review]

File: docs/get-started/getting-started-gcp.md (hunk 4 above)
Comment (zhanga5) [reply docs/get-started/getting-started-gcp.md]: refined

File: docs/get-started/getting-started-gcp.md (hunk 5 above)
Comment (zhanga5) [reply docs/get-started/getting-started-gcp.md]: refined

File: docs/get-started/getting-started-gcp.md (hunk 6 above)
Comment (zhanga5) [reply docs/get-started/getting-started-gcp.md]: refined

File: docs/get-started/getting-started-gcp.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhanga5) [reply docs/get-started/getting-started-gcp.md:206]: refined based on new requirement

**Review by GaryShen2008 — APPROVED** [review]

**Comment by GaryShen2008** [comment]
build

**Pull request merged**